fuse/all.py

```diff
@@ -43,6 +43,7 @@ def fuse_personnel():
         pd.read_csv(data_file_path("fuse/per_youngsville_pd.csv")),
         pd.read_csv(data_file_path("fuse/per_west_monroe_pd.csv")),
         pd.read_csv(data_file_path("fuse/per_carencro_pd.csv")),
+        pd.read_csv(data_file_path("fuse/per_post.csv")),
```

Comment:
Yeah, I wasn't sure how to treat it because POST is an external agency that allocates these certifications, as such, it is the only agency that can decertify an officer. Because it is an external agency, I could imagine a spectrum of responses taken by police departments with regard to officially documenting an officer's decertification. For example, I imagine that some departments choose to ignore the decertification by continuing to allow an officer to carry his firearm, while other departments might just move an officer to a department in which he/she does not need a firearm. That said, I do think that it makes more sense to treat it as we've been treating the other POST events. Let me know if you still agree and I'll go ahead and make those changes.

Reply:
Yeah, I think data from POST should be split across agencies. That's what our pipeline knows how to process. In case an officer is assigned to a different agency after decertification, it is still better to deal with that in our cross-agency matching script instead.